installer-clusters.cfg

```diff
@@ -1 +1 @@
-mainnet staging testnet
+nightly
```

Comment:
We'll want to add selfnode here as well I think.

Reply:
yep, that sounds like a good idea